### PR TITLE
映射文档 添加部分 inplace api PART.01

### PR DESCRIPTION
**Pull request by RedContritio**
添加部分 inplace api，其中 2 个 `nn.functional` api，20 个 `torch.Tensor` method。

> - [x] torch.nn.functional.hardtanh_
> - [x] torch.nn.functional.leaky_relu_
> - [x] torch.Tensor.abs_
> - [x] torch.Tensor.bitwise_not_
> - [x] torch.Tensor.bitwise_and_
> - [x] torch.Tensor.bitwise_or_
> - [x] torch.Tensor.bitwise_xor_
> - [x] torch.Tensor.cumprod_
> - [x] torch.Tensor.cumsum_
> - [x] torch.Tensor.detach_
> - [x] torch.Tensor.digamma_
> - [x] torch.Tensor.divide_
> - [x] torch.Tensor.eq_
> - [x] torch.Tensor.floor_divide_
> - [x] torch.Tensor.frac_
> - [x] torch.Tensor.gcd_
> - [x] torch.Tensor.ge_
> - [x] torch.Tensor.greater_equal_
> - [x] torch.Tensor.gt_
> - [x] torch.Tensor.greater_
> - [x] torch.Tensor.index_add
> - [x] torch.Tensor.index_put_

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.index_put_.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+torch.Tensor.index_put_(indices, value, accumulate=False)
+```
+
+### [paddle.Tensor.index_put_]()
+
+```python
+paddle.Tensor.index_put_(indices, value, accumulate=False)
+```
+
+其中 Pytorch 与 Paddle 参数有差异，具体如下：
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.index_put_.md]: 这个不太符合模板

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.index_put_.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.index_put_.md:18]: 这个是具体什么差异？分类是 `参数完全一致`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.hardtanh_.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+torch.nn.functional.hardtanh_(input, min_val=-1, max_val=1)
+```
+
+### [paddle.nn.functional.hardtanh_](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/hardtanh_cn.html)
+
+```python
+paddle.nn.functional.hardtanh_(x, min=-1.0, max=1.0)
+```
+
+Pytorch 相比 Paddle 支持更多其他参数，具体如下：
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.hardtanh_.md]: 分类是 `[仅参数名不一致]`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.leaky_relu_.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+## [仅参数名不一致]torch.nn.functional.leaky_relu_
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.leaky_relu_.md:1]: 分类是 [仅参数名不一致]

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.abs_.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+torch.abs_(input)
+```
+
+### [paddle.abs_](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/abs_cn.html)
+
+```python
+paddle.abs_(x)
+```
+
+Pytorch 相比 Paddle 支持更多其他参数，具体如下：
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.abs_.md]: 分类是 [仅参数名不一致]

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.index_put_.md (hunk 1 above)
Comment (RedContritio) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.index_put_.md]: solved

**Review by zhwesky2010 — APPROVED** [review]
LGTM

**Pull request merged**